charts/nfs-subdir-external-provisioner/values.yaml

```diff
@@ -2,7 +2,7 @@ replicaCount: 1
 strategyType: Recreate
 
 image:
-  repository: registry.k8s.io/sig-storage/nfs-subdir-external-provisioner
+  repository: k8s.gcr.io/sig-storage/nfs-subdir-external-provisioner
```

Comment:
According to https://kubernetes.io/blog/2023/03/10/image-registry-redirect/ we should use `registry.k8s.io` instead of `k8s.gcr.io`?

Reply:
yes, this change must be reverted